### DISCUSSION TITLE
[date-picker] 日期面板单独使用

### DISCUSSION THREAD

**Discussion by mengqi3691** [element-plus/element-plus#16376] [Feature Request]
Feature type: **`Other`**

## Details

- Enhanced component name: `date-picker`

## Description

希望日期面板，可以单独使用

### Additional comments

_(empty)_

**Comment by mengqi3691** [comment]
项目中有这个需求

**Comment by Dsaquel** [comment]
Released in `2.11.0`.
https://element-plus.org/en-US/component/date-picker-panel.html

**Comment by mengqi3691** [comment]
这是来自QQ邮箱的假期自动回复邮件。
 
您好，我最近正在休假中，无法亲自回复您的邮件。我将在假期结束后，尽快给您回复。
